code/ui/blocks/src/components/TableOfContents.tsx

```diff
@@ -139,14 +139,13 @@ export const TableOfContents = ({
       tocSelector: '.toc-wrapper',
       contentSelector: contentsSelector ?? '.sbdocs-content',
       headingSelector: headingSelector ?? 'h3',
-      ignoreSelector: ignoreSelector ?? '.skip-toc',
-      headingsOffset: 40,
-      scrollSmoothOffset: -40,
       /**
        * Ignore headings that did not
        * come from the main markdown code.
        */
-      // ignoreSelector: ':not(.sbdocs), .hide-from-toc',
+      ignoreSelector: ignoreSelector ?? '.docs-story *, .skip-toc',
```

Comment:
@shilman — This approach seems sound to me, but you're the one who left the commented-out version to seemingly accomplish the same thing, so thought you'd like to chime in.